### ISSUE TITLE
covid-19: Create "Assign to workflow step, workflow"

### ISSUE THREAD

**Issue by boulund** [ctmrbio/claritylims#148]
We will need a workflow that allows to assign specific samples to a certain workflow step (any workflow), in order to handle the backlog of SmiNet reporting.

The current plan:
* Create a workflow like "Create samples", that takes a list of samples and the name of the workflow+step that they samples should be assigned to

**Comment by chichackles** [comment]
File
https://github.com/Molmed/clarity-ext/blob/27230fe56cba4ec7169cf67686df167968714910/clarity_ext/data_cli.py
doing this from command line.
the usage is this:

clarity-data --to-file move-artifacts-plan lims-id-1 lims-id2 'SomeWorkflow' 'SomeStep' > 1.yml
# review the file, then execute:
clarity-data --to-file move-artifacts --commit 1.yml

**Comment by boulund** [comment]
@chichackles , did you leave this comment here for your own future reference with regards to this issue, or is this intended as instructions on how to use it? 

If the intention is to present usage instructions, I think it should be expanded a bit. We need to know which system user this needs to run as (I'm guessing `glsai`?), and if it has to run in a specific directory (I guess not?). There is also no information about the environment it requires (conda, or other dependencies). I'm guessing it expects the `clarity-ext` conda env, but I think we should document where that is located and how to activate it. Perhaps some examples of input files could be useful as well. 

Are you still working on creating a GUI workflow+step for this as we discussed yesterday?

**Comment by chichackles** [comment]
Yes, I have put it here for reference. I would like to have everything in one place. 
I hope to make it through GUI.

**Comment by boulund** [comment]
We moved all the affected samples manually using the `clarity-data` tool. Documentation in `/opt/gls/clarity/users/glsai/sminet_backlog_20200602` on prod.
